### ISSUE TITLE
Add `padding` shorthand helper function

### ISSUE THREAD

**Issue by markvandenbrink** [typestyle/csx#27]
https://developer.mozilla.org/en-US/docs/Web/CSS/padding

**Comment by trafnar** [comment]
I came here just to see if anyone was talking about this. I can imagine two ways for it to work:

Force all to same units, which is clean, and a common case
```javascript
px(5) // -> "5px"
px(5, 5) // -> "5px 5px"
```

Allow mix and match of units which is more verbose but flexible:
```javascript
padding(px(5), em(5)) // -> "5px 5em"
```

The thing is, the same logic applies for `padding` and `margin` and possibly other things, so having the exact same function definition for different names might be a bit strange, but I guess it is descriptive. Maybe there is a better name to the effect of `shorthand(px(5), em(5))`

**Comment by notoriousb1t** [comment]
It looks like we just need to setup an alias for margin/padding/etc. We already have a function for borderWidth which is pretty identical: https://github.com/typestyle/csx/blob/master/src/border.ts#L15

**Comment by notoriousb1t** [comment]
This has been added and released in [v9.1.0](https://github.com/typestyle/csx/releases/tag/v9.1.0)
